### DISCUSSION TITLE
dtk-template是否为GUI程序的样板？能否基于这个模板开发自己的DTK应用？

### DISCUSSION THREAD

**Discussion by 8MiYile** [linuxdeepin/developer-center#3752] [Q&A | 问答板块]
https://github.com/linuxdeepin/dtk-template

**Comment by Decodetalkers** [comment]
> https://github.com/linuxdeepin/dtk-template

这个更加类似库的模板，倒是确实应该提供一个widget的

**Reply by 8MiYile** [reply]
啊这，坐等widget的template，我好尝试摸界面来
